libs/utils/src/crashsafe.rs

```diff
@@ -5,6 +5,7 @@ use std::{
     io::{self, Write},
 };
 
+#[cfg(target_os = "linux")]
```

Comment:
perhaps move it inside 
    #[cfg(target_os = "linux")]
section? otherwise very unobvious why it is here